projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts

```diff
@@ -14,7 +14,9 @@ enum Tab {
   styleUrls: ['./protax-form.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ProtaxFormComponent {
+export class ProtaxFormComponent implements OnChanges {
+  @ViewChild('fileInput') fileInput: ElementRef;
```

Comment:
this could be typed to `ElementRef<HTMLInputElement>` so that the value used bellow wouldn't have to come from any

Reply:
type added